File: hw/Vaca/u1/HW09CDToJavaScripAndPhyton/ec.edu.espe.farmsimulator.view/chickenFarmSimulator.js
import { Chicken } from '../ec.edu.espe.farmsimulator.model/Chicken.js';
//const Chicken = import("../ec.edu.espe.farmsimulator.model/Chicken.js");

/*author:
    Luis Vaca, TheJavaBandits, DCCO-ESPE
*/

class ChickenFarmSimulator {
    static main() {
        console.log("Vaca's Chicken Farm Simulator V 0.5");

        let chicken1 = new Chicken(1, "Lucy", "brown", 1, true, new Date());
        console.log("Chicken 1 --> " + chicken1.toString());

        let id = 2;
        let name = "Maruja";
        let color = "black";
        let age = 1;
        let molting = false;
        let bornOnDate = new Date();

        let chicken2 = new Chicken(id, name, color, age, molting, bornOnDate);
        console.log("Chicken 2 --> " + chicken2.toString());
    }
}

ChickenFarmSimulator.main();

File: ws/segarra/u1/WS08CDToCode/ChickenFarmSimulator.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.chickenfarmsimulator.view;

import ec.edu.espe.chickenfarmsimulator.model.Chicken;
import java.util.Date;

/**
 * Chicken Farm Simulator View
 * 
 * Author: Eduardo Segarra, TheJavaBandits, DCCO-ESPE
 */
public class ChickenFarmSimulator {
    public static void main(String[] args) {
        System.out.println("Segarra's Chicken Farm Simulator V 0.5");
        
        Chicken chicken1 = new Chicken(1, "Lucy", "brown", 1, true, new Date());
        System.out.println("Chicken 1 --> " + chicken1);
        
        int id = 2;
        String name = "Maruja";
        String color = "black";
        int age = 1;
        boolean molting = false;
        Date bornOnDate = new Date();
        
        Chicken chicken2 = new Chicken(id, name, color, age, molting, bornOnDate);
        System.out.println("Chicken 2 --> " + chicken2);
    }
}